lib/oban/testing.ex

```diff
@@ -158,10 +158,19 @@ defmodule Oban.Testing do
   @doc since: "0.3.0"
   @spec assert_enqueued(repo :: module(), opts :: Keyword.t()) :: true
   def assert_enqueued(repo, [_ | _] = opts) do
-    assert get_job(repo, opts),
-           "Expected a job matching #{inspect(opts)} to be enqueued. Found: #{
-             inspect(available_jobs(repo, opts))
-           }"
+    job = Enum.into(opts, %{})
```

Comment:
This probably isn't the best variable name

Reply:
Maybe not, since it is a map and not a job struct. It could be done inline instead, to avoid naming it entirely:

```elixir
    #{inspect(Map.new(opts), pretty: true)}
```

Reply:
Okay, I like that :+1: